guides/CreateGINPrivateRepo.md

```diff
@@ -0,0 +1,276 @@
+
+## Other useful tips
+
+- To [unannex a file](https://gin.g-node.org/G-Node/Info/wiki/FAQ+Troubleshooting#how-to-unannex-files), aka remove a file from the GIN tracking before uploading:
+
+  ```
+  gin git annex unannex [path/filename]
+  ```
+
+- To stop tracking the GIN repo locally delete the `.git` directory
+
+- To delete a GIN repository but keep the git repo:
+
+  - delete the repository in the GIN server via the browser
+  - delete the GIN local repository with `git annex uninit`
+    - this command removes relevant bits in `.git/annex` and `.git/objects`, but some pre-commits may need to be edited by hand (see this [SO post](https://stackoverflow.com/questions/24447047/remove-git-annex-repository-from-file-tree)).
+
+## File locking
+
+[File locking](https://gin.g-node.org/G-Node/Info/wiki/GIN+CLI+Usage+Tutorial#file-locking) is an important point in GIN repos and git-annex which surprisingly comes up quite late in the docs. Below the main ideas behind it.
+
+- Files in a GIN repo can be locked or unlocked.
+
+- The lock state of a file is persistent. This means that if I clone a GIN repo whose files are unlocked, I lock them in my local copy, and then upload that to the GIN server, the next time someone clones from the GIN repo the files they fetch will be locked.
+
+- The lock state relates to the nature of the placeholder file (aka, the file in the working directory when we do `gin get <repository`):
+
+  - on Unix-like systems: if a file is locked, its corresponding placeholder file will be a symlink pointing to the annexed content (i.e., the content under `.git/annex/objects`). This way we can open and inspect the file but not modify it. If a file is unlocked, the placeholder file in the working directory is an ASCII text file with a path. This path is _approximately_ where the content of the file will be downloaded to when we request it.
+  - on Windows: from the docs, if a file is locked, the placeholder file is a plain text file pointing to the content in the git annex. If a file is unlocked, presumably the behaviour is the same as in Unix-like systems. I haven't tested the situation on Windows.
+
+- Unlocked files can be edited. If the data is unlocked and the full content of the dataset is downloaded locally, the file in the working directory has content, and so does its copy under git annex. This doubles disk usage of files checked into the repo, but in exchange users can modify and revert files to previous commits.
+
+- Locked files cannot be edited. In MacOS, we may be prompted to unlock the files if we try to edit them, but we won't be able to save any changes because we don't have writing permissions. Files need to be committed before locking (double check).
```

Comment:
I don't understand.

On MacOS who prompts you to unlock when you try to edit? Is it a GIN tool?